### ISSUE TITLE
As a CIF user, I want to validate email as a unique identifier of a contact, so that I don't duplicate contact information

### ISSUE THREAD

**Issue by suhafa** [bcgov/cas-cif#694]
#### Description:
[Wireframe:](https://www.figma.com/file/PzOY8RyQnVXnjZJdSv2AqQ/Wireframing_Figma?node-id=1982%3A2664) ![image.png](https://images.zenhubusercontent.com/6201b10c9827341d8d5cbe9c/42074fc6-2eaf-4e3a-a337-be5b9e9ca8c5)
Use of email address as a unique identifier for each contact so as to not have repetition or duplicate contacts. This can enable user to know if they've previously entered the information, by a error message below the field. 

#### Acceptance Criteria:

Given I am a CIF Project Coordinator
And I am entering contact information on the contact form
When I am entering a contact email
And it already exists in the system
Then an error message is displayed

Given I am a CIF Project Coordinator
And I am editing contact information on the contact form
When I am updating a contact email
And it already exists in the system
Then an error message is displayed

#### Development Checklist:

- [x] Validation on email address - This can be done in the same way we validate the uniqueness of the proposal reference (using the `isUniqueValue('email')` computed column on the contact form_change record)
- [x] Warning message if email address exists in system
- [ ] Meets the DOD

#### Notes:

-

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [x] User story is included
- [x] User role and type are identified
- [x] Acceptance criteria are included
- [x] Wireframes are included (if required)
- [x] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

**Comment by pbastia** [comment]
This

**Comment by matthieu-foucault** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Climate-Action-Secretariat-60ca4121764d710011481ca2/issues/bcgov/cas-cif/694?planning-poker) with ZenHub @BCerki @dleard @gurjmatharu @mikevespi @pbastia @Sepehr-Sobhani

**Comment by LindsayMacfarlane** [comment]
Updated priority to high as this user story should be completed before soft launch to support initial data entry of project information.
